packages/schemas/src/db-entries/setting.ts

```diff
@@ -0,0 +1,40 @@
+// THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+
+import { z } from 'zod';
+
+import {
+  AdminConsoleConfig,
+  adminConsoleConfigGuard,
+  GeneratedSchema,
+  Guard,
+} from '../foundations';
+
+export type SettingDBEntry = {
+  id: string;
+  customDomain?: string | null;
```

Comment:
@simeng-li @wangsijie this looks so strange. why did we [added `null`](https://github.com/logto-io/logto/pull/154/files#diff-1e5890cf370a2c7b7078fa4e8f9b8b99ed84a6d753005a9fa33b8a6a8372032aR17) for optional fields? to me `undefined` already did the job.
as i practiced before, we can limit the only purpose of `null` is to "set empty explicitly" (e.g. in API body), would be much simpler